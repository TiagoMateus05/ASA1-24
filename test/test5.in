15 50
11 5 11 11 14 11 10 11 1 15 5 6 5 4 4
2 8 5 15 10 9 9 4 15 7 15 7 15 6 7
11 8 7 15 6 13 4 6 5 8 10 12 15 12 8
3 8 11 11 5 13 14 15 2 4 7 9 12 5 9
9 13 11 8 8 1 8 7 2 11 1 2 13 6 7
14 6 7 1 12 6 2 3 10 1 8 10 6 9 15
6 10 5 11 10 10 12 15 2 4 7 13 7 7 6
1 13 5 10 7 5 6 14 15 11 6 13 10 6 11
5 6 10 2 3 8 11 3 4 2 10 7 7 2 6
1 13 1 12 13 12 2 6 8 14 10 5 10 1 12
4 8 5 3 11 3 9 5 4 6 4 15 10 3 14
1 6 1 9 12 5 5 1 9 3 2 8 12 9 1
5 15 11 13 12 4 6 11 14 12 11 12 3 13 4
11 15 9 1 9 6 13 4 6 10 12 5 3 15 8
6 8 11 9 14 15 15 12 2 5 14 11 14 12 9
9 5 6 14 12 3 14 11 6 8 4 11 2 8 12 2 5 9 7 10 8 8 13 6 3 11 15 5 15 4 5 10 1 3 2 15 8 7 10 3 2 7 8 10 15 1 13 6 4 10
2