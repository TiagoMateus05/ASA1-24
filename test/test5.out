1
(((((((((((((((((((((((((((((((((((((((((((((9 5) 6) 14) 12) 3) 14) 11) 6) 8) 4) 11) 2) 8) 12) 2) 5) 9) 7) 10) 8) 8) 13) 6) 3) 11) 15) 5) ((15 4) 5)) 10) 1) (3 2)) 15) 8) 7) 10) (3 2)) 7) 8) 10) 15) 1) 13) 6) 4) 10)
